docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -169,11 +179,13 @@ Types of instruments currently available:
 - For timing things, Histogram is usually preferred. Often it's useful to understand the tail of these distributions (90th, 95th, 99th percentile) rather than averages or
   totals.
 
-- Other common cases, such as business metrics, physical sensors, cache hit rates, or sizes of caches, queues, and files are usually well suited for `ObservableGauge`.
+- Other common cases, such as business metrics, physical sensors, cache hit rates, or sizes of caches, queues, and files are usually well suited for `UpDownCounter` or `ObservableUpDownCounter`.
```

Comment:
Cache hit rates might fit `ObservableGauge` (probably the same for many physical sensors and business metrics).

Reply:
I agree that ObservableGauge would work for those scenarios, but I intentionally didn't advertise it as a 2nd recommended option to make the overall guidance simpler and more concise. If you think we should be talking about it more happy to discuss.

Reply:
Maybe consider "Other common cases, such as size of caches, queues and files are usually ..."? (in this way folks who model physical sensors such as temperature / humidity / air pressure won't find it surprising/misleading)

Reply:
Yes, delete both "business metrics" and "physical sensors" (because most of these won't fit for UpDownCounter/ObservableUpDownCounter), OR keep them and mention "ObservableGauge".

Reply:
Done : )